### PR TITLE
Add tests/cypress config

### PR DESCRIPTION
**Pull request by JohnMcLear**

### CODE REVIEW

**Comment by JohnMcLear** [comment]
Just adding last files now..

**Comment by JohnMcLear** [comment]
Ready

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
   },
   "devDependencies": {
     "eslint": "^7.20.0",
+    "eslint-plugin-cypress": "^2.11.2",
```
Comment (rhansen) [inline package.json:42]: Also add it to `peerDependencies`. Technically it doesn't need to be in `devDependencies`, but we might as well keep the two in sync.

File: README.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ Etherpad plugins so that your code stays consistent with the Etherpad codebase.
         eslint-plugin-prefer-arrow \
         eslint-plugin-promise \
         eslint-plugin-you-dont-need-lodash-underscore \
+        eslint-plugin-cypress \
```
Comment (rhansen) [inline README.md:36]: Please keep this list sorted.

**Review by rhansen — COMMENTED** [review]
Please create `tests/cypress.js` that extends `../tests.js` and enables the plugin with recommended rules.

**Comment by rhansen** [comment]
I pushed a commit for squash. If it looks OK to you feel free to merge.

**Pull request merged**